site/en/workflows.md

```diff
@@ -1280,21 +1492,6 @@ page](https://ocr-d-repo.scc.kit.edu/api/v1/dataresources/dda89351-7596-46eb-973
       <td>ocrd-tesserocr-deskew</td>
```

Comment:
Gone in the command list below, but still here! (Cf. [my comment last time](https://github.com/OCR-D/ocrd-website/pull/189#discussion_r536154337) …)

You need to remove region-level deskewing here, too, and renumber from 9 upwards.